<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Driver_ID_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Driver's ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Routes_assigned">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Route 17628 ( start to destination)</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="Stops_table">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Stop Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Reaching Time</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Johar</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>8:00 am</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Saddar</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>8:15 am</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Buses_assigned">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bus 703</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Reached_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reached</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menudriver_s_screen">
    <property name="title">
     <string>driver's screen</string>
    </property>
   </widget>
   <addaction name="menudriver_s_screen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
